src/native/Icon/index.js

```diff
@@ -8,8 +8,8 @@ import iconsMap from './icons.json';
 
 export type Props = {|
   +name: string,
-  +size?: number,
-  +color?: string,
+  size?: number,
```

Comment:
I had to do this when cloning an Icon component and flow would rightly complain that I am overriding read-only props otherwise.